src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -110,6 +131,48 @@ public static String getCurrentBallerinaVersion() {
         }
     }
 
+    /**
+     * Sets custom SSL context for update tool.
+     */
+    public static void setCustomSSLContext() throws IOException {
+        // Load custom truststore if provided, otherwise use the default truststore
+        try {
+            KeyStore truststore;
+            if (trustStorePath != null && trustStorePassword != null) {
+                truststore = KeyStore.getInstance(KeyStore.getDefaultType());
```

Comment:
Can we move `truststore = KeyStore.getInstance(KeyStore.getDefaultType());` line out of the if/else since this is common to both flows?

Reply:
Addressed review suggestions